### ISSUE TITLE
Unable to start SonarQube v9.2.0.49834

### ISSUE THREAD

**Issue by blazejpawlak** [sbaudoin/sonar-yaml#63]
I just tried updating our instance of SonarQube to latest available 9.2.0.49834 and after database migration it crashes with the following stack trace. 

```
2021.11.22 19:27:55 ERROR web[][o.s.s.p.Platform] Background initialization failed. Stopping SonarQube
java.lang.IllegalStateException: Fail to load plugin Configuration detection fot Code Quality and Security [config]
	at org.sonar.server.plugins.ServerExtensionInstaller.installExtensions(ServerExtensionInstaller.java:77)
	at org.sonar.server.platform.platformlevel.PlatformLevel4.start(PlatformLevel4.java:581)
	at org.sonar.server.platform.PlatformImpl.start(PlatformImpl.java:213)
	at org.sonar.server.platform.PlatformImpl.startLevel34Containers(PlatformImpl.java:187)
	at org.sonar.server.platform.PlatformImpl$1.lambda$doRun$0(PlatformImpl.java:120)
	at org.sonar.server.platform.PlatformImpl$AutoStarterRunnable.runIfNotAborted(PlatformImpl.java:370)
	at org.sonar.server.platform.PlatformImpl$1.doRun(PlatformImpl.java:120)
	at org.sonar.server.platform.PlatformImpl$AutoStarterRunnable.run(PlatformImpl.java:354)
	at java.base/java.lang.Thread.run(Thread.java:829)
Caused by: java.lang.IllegalStateException: Unable to register extension org.sonar.api.config.PropertyDefinition from plugin 'config'
	at org.sonar.core.platform.ComponentContainer.addExtension(ComponentContainer.java:243)
	at org.sonar.server.plugins.ServerExtensionInstaller.installExtension(ServerExtensionInstaller.java:85)
	at org.sonar.server.plugins.ServerExtensionInstaller.installExtensions(ServerExtensionInstaller.java:69)
	... 8 common frames omitted
Caused by: org.picocontainer.PicoCompositionException: Duplicate Keys not allowed. Duplicate for 'org.sonar.api.config.PropertyDefinition-sonar.yaml.file.suffixes'
	at org.picocontainer.DefaultPicoContainer.addAdapterInternal(DefaultPicoContainer.java:438)
	at org.picocontainer.DefaultPicoContainer.addAdapter(DefaultPicoContainer.java:459)
	at org.picocontainer.DefaultPicoContainer.addComponent(DefaultPicoContainer.java:556)
	at org.picocontainer.DefaultPicoContainer.access$300(DefaultPicoContainer.java:84)
	at org.picocontainer.DefaultPicoContainer$AsPropertiesPicoContainer.addComponent(DefaultPicoContainer.java:1149)
	at org.sonar.core.platform.ComponentContainer.addExtension(ComponentContainer.java:241)
	... 10 common frames omitted
```

Anything someone from here can help with?

**Comment by sbaudoin** [comment]
Hello,

The plugin is known not to be compatible with SQ 9.2, see #58 (the PR was for SQ 9.1 because this very version was supposed to bring native support for YAML but this was actually postponed to 9.2 for whatever reason).

From your message I understand that SQ 9.2 has been recently released: I'm going to update the PR for SQ 9.2 then.

**Comment by sbaudoin** [comment]
Version 1.7.0 has just been released
